fuse/losses/segmentation/loss_dice.py

```diff
@@ -65,7 +65,7 @@ def __call__(self, predict, target):
         predict = predict.contiguous().view(predict.shape[0], -1)
         target = target.contiguous().view(target.shape[0], -1)
 
-        if target.dtype == torch.int64:
+        if target.dtype == torch.int64 or target.dtype == torch.int32:
```

Comment:
Why do we care about the type?
Can we convert it anyway?

Reply:
I also think that the to device is not necessary here - can you remove it>

Reply:
I remember I had to add this option. not sure what was the case.
I removed both the to(device) .